README.md

```diff
@@ -37,6 +37,14 @@ Install KeePassX from the Red Hat (or CentOS) repository:
 sudo yum install keepassx
 ```
 
+### Arch Linux
+
+Install KeePassX from source via AUR, [keepassx-git](https://aur.archlinux.org/packages/keepassx-git).
+
+```bash
+yaourt -S keepassx-git
```

Comment:
Now everyone is using yaourt. Please don't suggest yaourt as an AUR helper. Archlinux users that use AUR already know how to install it so providing a bash solution which uses yaourt is not necessary.